Error "undefined: GetInvokingContext" for Go target

<!--
Before submitting an issue to ANTLR, please check off these boxes:
 
- [ X] I am not submitting a question on how to use ANTLR; instead, go to [antlr4-discussion google group](https://groups.google.com/forum/#!forum/antlr-discussion) or ask at [stackoverflow](http://stackoverflow.com/questions/tagged/antlr4)
- [ X] I have done a search of the existing issues to make sure I'm not sending in a duplicate

Please include information about the expected behavior, actual behavior, and the smallest grammar or code that reproduces the behavior. If appropriate, please indicate the code generation targets such as Java, C#, ...  Pointers into offending code regions are also very welcome.
-->

Test.g4:
```g4
grammar Test;

itemList locals [ []string Items ]
   : '<' item+ '>' EOF { fmt.Printf("Items in list: %v\n", $Items) }
   ;

item
   : tkn='A' { $itemList::Items = append($itemList::Items, $tkn.GetText()) }
   ;

// Ignore whitespace in lexer
WhiteSpace: [ \r\n\t]+ -> skip;
```

Go code for testing:
```go
package main

import (
	"github.com/antlr/antlr4/runtime/Go/antlr"
	"tester/parser"
)

type testListener struct {
	*parser.BaseTestListener
}

func main() {
	is := antlr.NewInputStream("< A A A >")

	lexer := parser.NewTestLexer(is)
	stream := antlr.NewCommonTokenStream(lexer, antlr.TokenDefaultChannel)

	p := parser.NewTestParser(stream)
	antlr.ParseTreeWalkerDefault.Walk(&testListener{}, p.Start())
}
```

Go module descriptor for compiling the code above:
```
module tester

go 1.12

require github.com/antlr/antlr4 v0.0.0-20200503195918-621b933c7a7f
```

Generating the parser:
```bash
java -jar $PWD/antlr-4.8-complete.jar -Dlanguage=Go -o parser Test.g4
```

Running the code tester:
```bash
$ go run tester.go
# tester/parser
parser/test_parser.go:328:2: undefined: GetInvokingContext
```

--- a/tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg
+++ b/tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg
@@ -906,8 +906,8 @@ ThisRulePropertyRef_text(r) ::= "p.GetTokenStream().GetTextFromTokens(localctx.G
 ThisRulePropertyRef_ctx(r) ::= "<ctx(r)>"
 ThisRulePropertyRef_parser(r) ::= "p"
 
-NonLocalAttrRef(s) ::= "GetInvokingContext(<s.ruleIndex>).<s.escapedName>"
-SetNonLocalAttr(s, rhsChunks) ::= "GetInvokingContext(<s.ruleIndex>).<s.escapedName> = <rhsChunks>"
+NonLocalAttrRef(s) ::= "p.GetInvokingContext(<s.ruleIndex>).(*<s.ruleName; format={cap}>Context).<s.escapedName>"
+SetNonLocalAttr(s, rhsChunks) ::= "p.GetInvokingContext(<s.ruleIndex>).(*<s.ruleName; format={cap}>Context).<s.escapedName> = <rhsChunks>"
 
 AddToLabelList(a) ::= "<ctx(a.label)>.<a.listName> = append(<ctx(a.label)>.<a.listName>, <labelref(a.label)>)"
 

